doc/changelog/1.25.0-changelog.rst

```diff
@@ -563,7 +563,7 @@ A total of 530 pull requests were merged for this release.
 * `#23565 <https://github.com/numpy/numpy/pull/23565>`__: DOC: Add directive for C-type ``PyArrayMapIterObject``
 * `#23566 <https://github.com/numpy/numpy/pull/23566>`__: CI: disable intel_spr_sde_test for now
 * `#23568 <https://github.com/numpy/numpy/pull/23568>`__: MAINT: remove usages of sys.exc_info
-* `#23569 <https://github.com/numpy/numpy/pull/23569>`__: TST: try readding test_new_policy on musl
+* `#23569 <https://github.com/numpy/numpy/pull/23569>`__: TST: try re-adding test_new_policy on musl
```

Comment:
oh -- missed this one, was probably modified by another than "fix all" commit... will fix up now...